BUILDGUIDE.md

```diff
@@ -242,6 +242,12 @@ To use this feature, you must enable the following AppContext switch at applicat
 
 **"Switch.Microsoft.Data.SqlClient.EnableRetryLogic"**
 
+## Enabling row version null behaviour
+
+`SqlDataReader` returns a `DBNull` value instead of an empty `byte[]`. To active the legacy behaviour, you must enable the following AppContext switch that is cached on first check for the entire process:
+
+**"Switch.Microsoft.Data.SqlClient.LegacyRowVersionNullBehaviour"**
```

Comment:
Is the Switch `LegacyRowVersionNullBehaviour` or `LegacyRowVersionNullBehavior`

Reply:
It's actually `LegacyRowVersionNullBehaviour` https://github.com/dotnet/SqlClient/pull/998
@David-Engel any concerns over UK/US grammar?

Reply:
We should use US grammar.

Reply:
Boooo, but ok i understand.